extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRecorder.java

```diff
@@ -199,7 +201,7 @@ private TenantConfigContext createTenantContext(Vertx vertx, OidcTenantConfig oi
             options.setProxyOptions(proxyOpt.get());
         }
 
-        if (oidcConfig.tls.verification == Verification.NONE) {
+        if (oidcConfig.tls.verification == Verification.NONE || tlsConfig.trustAll) {
```

Comment:
@glefloch Hi, in this specific example, should it be `&&` ? (assuming `tlsConfig.trustAll` is `true` by default)

IMHO, keeping the extension specific properties is OK because it can let the users be specific about the concrete extensions, but hence the extension specific setting should take the priority ?

Reply:
@sberyozkin yes you are right, I'm going to update this.

Reply:
@glefloch So as you reminded `trustAll` is `false` by default, then if want to prioritize on the extension specific property then if we remove the default values for `oidcConfig.tls.verification` (and similarly for other extensions) then it should work
```
boolean trustAll = oidcConfig.tls.verification.isPresent() ? oidcConfig.tls.verification == Verification.NONE : tlsConfig.trustAll;
```

What do you think ?

Reply:
Yes indeed, I will give it a try. That seem to be a better approach. Thanks